ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -337,6 +337,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
```

Comment:
I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.

Reply:
@jianlinliu Just to confirm, do you want me to remove following job `azure-ipi-private-p2-f14` in this PR?

Reply:
@JianLi-RH per my understanding, yes, you're right.

Reply:
right